STORE_VARIABLES.md

```diff
@@ -16,6 +16,7 @@
 | `settings-discord-rich-presence`           | Boolean | `true` or `false`            | `false`                      | Integration with Discord Rich Presence                         |
 | `settings-start-on-boot`                   | Boolean | `true` or `false`            | `false`                      | Open player when system starts                                 |
 | `settings-start-minimized`                 | Boolean | `true` or `false`            | `false`                      | Open player minimized in tray                                  |
+| `settings-lyrics-always-top`               | Boolean | `true` or `false`            | `false`                      | Open player minimized in tray                                  |
```

Comment:
Just notice that this need to be updated as well.

Reply:
On it

Reply:
Also going to sort the settings alphabetically because its really irritating me